docs/core/messages.md

```diff
@@ -74,3 +74,8 @@ func (msg MsgIssue) GetSigners() []sdk.Address {
 }
 ```
 
+Message logic is only handled by the `DeliverTx()` function of a transaction. For example,
```

Comment:
These are helpful, but we need to find the right place for them now. Have you seen the new docs structure: https://github.com/cosmos/cosmos-sdk/tree/develop/docs ? 

We could add these notes to the ABCI section there for now